index.d.ts

```diff
@@ -0,0 +1,48 @@
+declare module '@pusher/push-notifications-web' {
+  export interface TokenProviderResponse {
+    token: string;
+  }
+  export interface ITokenProvider {
```

Comment:
It would be nice if this interface could just be `TokenProvider` and then we could have a specific name for the provided token provider that implements it.

Reply:
Can't do that without a breaking change, do you think it's worth doing now?
I know we mentioned changing the class name to BeamsTokenProvider

Reply:
I don't feel strongly either way. It could stay the way it is for now and we could change it with other breaking changes later.